config/v1/types.go

```diff
@@ -1,8 +1,11 @@
 package v1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+
+	buildv1 "github.com/openshift/api/build/v1"
```

Comment:
I wouldn't do this for the simple type you're pulling in.  It's a large dep fanout and no consumer of this API gains a benefit from the shared type.  Customers will use json/yaml, configobserver is required to serialize manually into unstructured.